.github/workflows/3x.yml

```diff
@@ -9,10 +9,7 @@ on:
         description: Run arbitrary functional tests group by name or all (short and long) tests (default)
         required: true
         default: all
-      tmate_enabled:
-        description: Enable tmate debugging
-        required: true
-        default: "0"
+
```

Comment:
We want to keep tmate.  I know kevin commented as well, but there are a number of removals related to it, so I'm commenting as well.